src/main/java/controller/RacingController.java

```diff
@@ -1,18 +1,26 @@
 package controller;
 
+import model.Car;
 import model.RacingGame;
+import model.RacingRecord;
 import view.InputView;
 import view.ResultView;
 
+import java.util.List;
+
 public class RacingController {
     public void run() {
         InputView inputView = new InputView();
 
-        int numberOfCars = inputView.askNumberOfCars();
+        List<Car> cars = inputView.askCarNames();
         int racingTurn = inputView.askRacingTurn();
 
-        RacingGame racingGame = new RacingGame(numberOfCars);
-        ResultView resultView = racingGame.run(racingTurn);
+        RacingGame racingGame = new RacingGame(cars);
+        RacingRecord racingRecord = racingGame.run(racingTurn);
+        racingRecord.saveWinners(racingGame.winnerNames());
+
+        ResultView resultView = new ResultView(racingRecord);
+        resultView.draw();
         resultView.print();
```

Comment:
```suggestion
        RacingGame racingGame = new RacingGame(cars);
        RacingRecord racingRecord = racingGame.run(racingTurn);

        ResultView resultView = new ResultView();
        resultView.draw(racingRecord);
```

레코드로 부터 우승자를 꺼내올 수 있을 것 같습니다.